pkg/apis/wildfly/v1alpha2/wildflyserver_conversion.go

```diff
@@ -0,0 +1,4 @@
+package v1alpha2
+
+// Hub marks this type as a conversion hub.
+func (*WildFlyServer) Hub() {}
```

Comment:
iirc, the Hub is in package `v1alpha2` so I'd expect that the corresponding converter method is in the same package

Reply:
In this case we selected v1alpha2 as the implementation of the Hub interface, so we do not need to add any conversion here.